agent/engine/docker_task_engine_test.go

```diff
@@ -423,7 +423,7 @@ func TestSteadyStatePoll(t *testing.T) {
 	// Two steady state oks, one stop
 	gomock.InOrder(
 		client.EXPECT().DescribeContainer("containerId").Return(api.ContainerRunning, DockerContainerMetadata{DockerId: "containerId"}).Times(2),
-		client.EXPECT().DescribeContainer("containerId").Return(api.ContainerStopped, DockerContainerMetadata{DockerId: "containerId"}),
+		client.EXPECT().DescribeContainer("containerId").Return(api.ContainerStopped, DockerContainerMetadata{DockerId: "containerId"}).AnyTimes(),
```

Comment:
Yeah, I understand. let's add that to the TODO then. We can address that in a different PR.